TCD decoupling

nickpalumbo8 opened the pull request:
Visualized configuration data of new STL models.
Created a Cluster Configuration File (CCF) to allow for the clusters and thrusters to be rotated independently.
Also includes functions to read the file and edits to source code to account for the clusters position in the position of the plumeMesh and plume strike, changing the Thruster Configuration File (TCF) to be with respect to the CCF.

andytorrestb reviewed (changes requested):
1. Please make sure all data.txt files have heritage format. 
2. Please update PR description to clarify this is for STL visualization.
3. Why do the file names include the word "decoupled"? can we remove this?
4. Can you please include a call to graph_cluster in your test case? It is unclear when and how it is used.

nickpalumbo8 commented:
1. Fixed.
2. Updated.
3. Removed.
4. Please see the commit "RPOD.py changes", details of where and when graph_clusters is called is described there.

andytorrestb commented:
lets re-open after MVP is running.
